assembl/tests/fixtures/graphql.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 from datetime import datetime
 import pytest
+from freezegun import freeze_time
```

Comment:
if you are using a new package please add to the requirements

Reply:
Done